main.js

```diff
@@ -0,0 +1,15125 @@
+                "manas",
+                "manat",
+                "mandi",
+                "maneb",
+                "maned",
+                "maneh",
+                "manes",
+                "manet",
+                "mangs",
+                "manis",
+                "manky",
+                "manna",
+                "manos",
+                "manse",
+                "manta",
+                "manto",
+                "manty",
+                "manul",
+                "manus",
+                "mapau",
+                "maqui",
+                "marae",
+                "marah",
+                "maras",
+                "marcs",
+                "mardy",
+                "mares",
+                "marge",
+                "margs",
+                "maria",
+                "marid",
+                "marka",
+                "marks",
+                "marle",
+                "marls",
+                "marly",
+                "marms",
+                "maron",
+                "maror",
+                "marra",
+                "marri",
+                "marse",
+                "marts",
+                "marvy",
+                "masas",
+                "mased",
+                "maser",
+                "mases",
+                "mashy",
+                "masks",
+                "massa",
+                "massy",
+                "masts",
+                "masty",
+                "masus",
+                "matai",
+                "mated",
+                "mater",
+                "mates",
+                "maths",
+                "matin",
+                "matlo",
+                "matte",
+                "matts",
+                "matza",
+                "matzo",
+                "mauby",
+                "mauds",
+                "mauls",
+                "maund",
+                "mauri",
+                "mausy",
+                "mauts",
+                "mauzy",
+                "maven",
+                "mavie",
+                "mavin",
+                "mavis",
+                "mawed",
+                "mawks",
+                "mawky",
+                "mawns",
+                "mawrs",
+                "maxed",
+                "maxes",
+                "maxis",
+                "mayan",
+                "mayas",
+                "mayed",
+                "mayos",
+                "mayst",
+                "mazed",
+                "mazer",
+                "mazes",
+                "mazey",
+                "mazut",
+                "mbira",
+                "meads",
+                "meals",
+                "meane",
+                "means",
+                "meany",
+                "meare",
+                "mease",
+                "meath",
+                "meats",
+                "mebos",
+                "mechs",
+                "mecks",
+                "medii",
+                "medle",
+                "meeds",
+                "meers",
+                "meets",
+                "meffs",

```

Comment:
these are the toasts upon game completion. GENIUS, PHEW, etc on win, solution word on loss. this is probably good place to set our new localStorage elements if you decvide to do it on close of game only

Reply:
I hooked the initial stats display which seems to work just fine. It won't run on restore from local storage but the result should be saved at that point